packages/calypso-e2e/src/lib/pages/cart-checkout-page.ts

```diff
@@ -107,7 +107,7 @@ export class CartCheckoutPage {
 	 */
 	async validatePaymentForm(): Promise< void > {
 		const cardholderNameLocator = this.page.locator( selectors.cardholderName );
-		await cardholderNameLocator.waitFor( { timeout: 20 * 1000 } );
+		await cardholderNameLocator.waitFor( { state: 'visible', timeout: 20 * 1000 } );
```

Comment:
FYI, the `visible` state is [the default one](https://playwright.dev/docs/api/class-locator#locator-wait-for:~:text=Defaults%20to%20%27visible%27) when calling `waitFor` so this should not have any effect here.